firefox_privacy_notice/fx56_notice.md

```diff
@@ -145,6 +141,7 @@ You can install Add-ons from addons.mozilla.org (“AMO”) or from the Firefox
 
 ---
 
+{: #pre-release }
```

Comment:
Please also add the following here, probably above:

```

### Footnote

```

It will not be displayed, it is used by me to denote the footnote from the rest of the content.